salt/sciencebeam-texture/config/home-deployuser-sciencebeam-texture-.env

```diff
@@ -1,4 +1,6 @@
 COMPOSE_PROJECT_NAME=sciencebeam-texture
 IMAGE_TAG={{ salt['elife.image_tag']() }}
 SCIENCEBEAM_TAG={{ salt["elife.image_label"]('elifesciences/sciencebeam_texture', 'org.elifesciences.dependencies.sciencebeam', salt["elife.image_tag"]()) }}
+SCIENCEBEAM_AUTOCUT_REPO=elifesciences/sciencebeam-gym-autocut_trained
+SCIENCEBEAM_AUTOCUT_TAG=40ff412896fc3b5077803c759beaf92f3e8970cb-elife-author-submitted-pdf-train-1000
```

Comment:
not sure if the dynamic tag used for `SCIENCEBEAM_TAG` applies for `SCIENCEBEAM_AUTOCUT`. The latter is probably more manually updated. (But as noted before, it would be good if it was in line with [sciencebeam-texture/](https://github.com/elifesciences/sciencebeam-texture)